src/components/LinksBox/LinksBox.module.scss

```diff
@@ -18,3 +18,11 @@
     list-style-position: inside;
   }
 }
+
+.darkMode {
```

Comment:
Suggest putting this in the generic css file other copy-pasting in the CSS file for each component